types/playlist.d.ts

```diff
@@ -25,6 +30,9 @@ export type PlaylistItem = {
   feedid: string;
   image: string;
   images: Image[];
+  shelfImage?: ImageData;
```

Comment:
I would really like to decouple this so that the delivery api interface and the app's internal data structure aren't mixed together, but probably a nice to have for another day...